src/main/java/where2eat.ump

```diff
@@ -0,0 +1,68 @@
+namespace ca.mcgill.ecse.where2eat.model;
+
+enum Category {Chinese, Indian, Portuguese}
+
+class Where2Eat {
+  1 <@>- * User users;
+  1 <@>- * Login loginIDs;
+  1 <@>- * Group groups;
+  1 <@>- * Restaurant restaurants;
+  1 <@>- * Location locations;
+  1 <@>- * Address addresses;
+  1 <@>- * PhoneNumber phoneNumbers;
+}
+
+class User
```

Comment:
Missing a role enumeration for User/Admin distinction. or some kind of Abstraction-Occurrence pattern. I'd prefer roles as an enumeration IMO because it simplifies thing a bit with JWT.